modules/security-container-content-external-scanning.adoc

```diff
@@ -0,0 +1,262 @@
+"low," or "health")
+|String
+
+|`data`
+|Data for this component (for example, count of vulnerabilities found or score)
+|String
+
+|`severityIndex`
+|Component index allowing for ordering and assigning graphical
+representation. The value is range `0..3` where `0` = low.
+|Integer
+
+|`reference`
+|URL of information source and/or more details. Optional.
+|String 
+|===
+
+[[security-example-annotation-values]]
+=== Example annotation values
+
+This example shows an OpenSCAP annotation for an image with
+vulnerability summary data and a compliance boolean:
+
+.OpenSCAP Annotation
+[source,json]
+----
+{
+  "name": "OpenSCAP",
+  "description": "OpenSCAP vulnerability score",
+  "timestamp": "2016-09-08T05:04:46Z",
+  "reference": "https://www.open-scap.org/930492",
+  "compliant": true,
+  "scannerVersion": "1.2",
+  "summary": [
+    { "label": "critical", "data": "4", "severityIndex": 3, "reference": null },
+    { "label": "important", "data": "12", "severityIndex": 2, "reference": null },
+    { "label": "moderate", "data": "8", "severityIndex": 1, "reference": null },
+    { "label": "low", "data": "26", "severityIndex": 0, "reference": null }
+  ]
+}
+----
+
+This example shows the
+link:https://catalog.redhat.com/software/containers/explore[Red Hat Container Catalog]
+annotation for an image with health index data
+with an external URL for additional details:
+
+.Red Hat Container Catalog Annotation
+[source,json]
+----
+{
+  "name": "Red Hat Container Catalog",
+  "description": "Container health index",
+  "timestamp": "2016-09-08T05:04:46Z",
+  "reference": "https://access.redhat.com/errata/RHBA-2016:1566",
+  "compliant": null,
+  "scannerVersion": "1.2",
+  "summary": [
+    { "label": "Health index", "data": "B", "severityIndex": 1, "reference": null }
+  ]
+}
+----
+
+[[security-annotating-image-objects]]
+== Annotating image objects
+
+While image stream objects
+are what an end-user of {product-title} operates against,
+image objects are annotated with
+security metadata. Image objects are cluster-scoped, pointing to a single image
+that may be referenced by many image streams and tags.
+
+[[security-example-annotate-CLI]]
+=== Example annotate CLI command
+
+Replace `<image>` with an image digest, for example
+`sha256:401e359e0f45bfdcf004e258b72e253fd07fba8cc5c6f2ed4f4608fb119ecc2`:
+
+----
+$ oc annotate image <image> \
+    quality.images.openshift.io/vulnerability.redhatcatalog='{ \
+    "name": "Red Hat Container Catalog", \
+    "description": "Container health index", \
+    "timestamp": "2020-06-01T05:04:46Z", \
+    "compliant": null, \
+    "scannerVersion": "1.2", \
+    "reference": "https://access.redhat.com/errata/RHBA-2020:2347", \
+    "summary": "[ \
+      { "label": "Health index", "data": "B", "severityIndex": 1, "reference": null } ]" }'
+----
+
+[[controlling-pod-execution]]
+== Controlling pod execution
+
+To programmatically control if an image may be run, the
```

Comment:
This is passive voice, suggest something instead like:

```
Use the 
`images.openshift.io/deny-execution` image policy to programmatically control if an image may be run.
```
Still unsure about the use of "may". Maybe clarify - does it mean "is allowed to be run"?

Reply:
Done. I also changed "may" to "can" since since you can do it if this is set. Whether or not we like it doesn't matter.